targets/targets.json

```diff
@@ -3029,6 +3029,31 @@
         "device_name": "STM32H743ZI",
         "bootloader_supported": true
     },
+    "UHURU_RAVEN": {
```

Comment:
Any storage components that should be added? Any SD card, external or internal flash?

Reply:
The RAVEN board has external SPI Flash.